src/components/TypeTwo/TitleSmallTwo.js

```diff
@@ -0,0 +1,12 @@
+import { TypeBase, TypeFoundry } from '../TypeBase/TypeBase.js'
+
+// Currently drastically different than existing TitleSmall for testing purposes
+export const TitleSmallTwo = {
+  Sans: {
+    Light300: TypeFoundry({
+      subtype: TypeBase.SUBTYPES.TITLE_MEDIUM,
+      typeface: TypeBase.TYPEFACES.THEINHARDT,
+      weight: TypeBase.WEIGHTS.MEDIUM_500,
+    }),
+  },
+}
```

Comment:
Purposefully very different from existing TitleSmall for testing purposes in consumer (CMS).